docs/updating/from_4.0/to_4.1.md

```diff
@@ -0,0 +1,131 @@
+---
+latest_tag: '4.1.1'
+---
+
+# Update from v4.0.x to v4.1
+
+This update procedure applies if you are using v4.0.latest.
+
+Go through the following steps to update to v4.1.
+
+## Update the app to v4.1
+
+First, run:
+
+=== "[[= product_name_content =]]"
+
+    ``` bash
+    composer require ibexa/content:[[= latest_tag =]] --with-all-dependencies --no-scripts
+    ```
+
+=== "[[= product_name_exp =]]"
+
+    ``` bash
+    composer require ibexa/experience:[[= latest_tag =]] --with-all-dependencies --no-scripts
+    ```
+
+=== "[[= product_name_com =]]"
+
+    ``` bash
+    composer require ibexa/commerce:[[= latest_tag =]] --with-all-dependencies --no-scripts
+    ```
+
+Continue with updating the app:
+
+=== "[[= product_name_content =]]"
+
+    ``` bash
+    composer recipes:install ibexa/content --force -v
+    ```
+
+=== "[[= product_name_exp =]]"
+
+    ``` bash
+    composer recipes:install ibexa/experience --force -v
+    ```
+
+=== "[[= product_name_com =]]"
+
+    ``` bash
+    composer recipes:install ibexa/commerce --force -v
+    ```
+
+The `recipes:install` command installs new YAML configuration files. Look through the old YAML files and move your custom configuration to the relevant new files.
+
+## Update the database
+
+[[% include 'snippets/update/db/db_backup_warning.md' %]]
+
+Apply the following database update script:
+
+### Ibexa DXP
+
+=== "MySQL"
+    ``` bash
+    mysql -u <username> -p <password> <database_name> < vendor/ibexa/installer/upgrade/db/mysql/ibexa-4.0.0-to-4.1.0.sql
+    ```
+
+=== "PostgreSQL"
+
+    ``` bash
+    psql <database_name> < vendor/ibexa/installer/upgrade/db/postgresql/ibexa-4.0.0-to-4.1.0.sql
+    ```
+
+## Configure the Product Catalog [[% include 'snippets/experience_badge.md' %]]
```

Comment:
```suggestion
## Configure the product catalog
```
I think, as both catalog and price engine are available in the content edition.
And lowercase, as elsewhere in the doc: https://doc.ibexa.co/en/master/guide/catalog/catalog/